tests/FeaturesTest.php

```diff
@@ -37,6 +38,10 @@ public function testFeatures(string $file): void
 
     private function analyze(string $file): int
     {
+        if (Str::contains($file, 'Features/Laravel8') && version_compare(app()->version(), '8.0.0') < 0) {
```

Comment:
I think you can do `version_compare(app()->version(), '8.0.0', '<')` here

Reply:
Yes but I thought it was less readable but as you wish

Reply:
IMO, `version_compare(...) < 0` is more confusing. Why do we expect it to be less than 0? (If you don't know the return types of version_compare) :smile:

Reply:
Most parts of PHP is confusing!